### ISSUE TITLE
Mock coding interviews in class

### ISSUE THREAD

**Issue by davidsng** [ga-students/wdi-sg-organisers#35]
For the benefit of those who need early preparation/practice, would it be helpful to have regular coding interview exercises in class while we still have Seb with us? It will take multiple practices to get it right.

**Comment by cbas** [comment]
:+1:

**Comment by rachlim** [comment]
@cbas yes it's technically a part of the outcomes programme which will happen during class- best to liase with Aziza on when (it could maybe take up one of her outcomes slots?) I'm also talking to a guy who's organising Coding Interview meet ups, will keep you guys posted on that!

**Comment by zellwk** [comment]
:+1: This would be an awesome thing to have.

**Comment by davidsng** [comment]
Even 30mins of peer coding interview every other day at the end of the day will do wonders to prep for the interview. Such time-pressured hackathons can also help recap things covered in class. If it's effective, @jsstrn and @lanbau can also make this a regular feature of WDI. 

For best improvement results, the guys ought to practice it much more regularly than just once or twice as per the outcome sessions. Such interviews are not natural. I've personally had to coach many candidates in consulting case interviews and they really only "got it" after a long time.

**Comment by jsstrn** [comment]
There's [HackerRank](http://hackerrank.com/) where you can solve coding problems.

**Comment by cbas** [comment]
@sevresbabylone For reference: We should hold the code breakfast tomorrow (2016-01-21) in the mock interview format. Pairing with one person as the interviewer and another as the candidate.

**Comment by caalberts** [comment]
[More coding problems](https://codility.com/programmers/lessons/18/)
